roles/generate_kubernetes_config/templates/panda-pulse.yaml.j2

```diff
@@ -0,0 +1,43 @@
+# {{ ansible_managed }}
+
+panda-pulse:
+  fullnameOverride: "panda-pulse"
+
+  image:
+    repository: {{ default_tooling_images.panda_pulse.split(':') | first }}
+    tag: {{ default_tooling_images.panda_pulse.split(':') | last }}
+    pullPolicy: {% if "latest" in (default_tooling_images.panda_pulse.split(':') | last) %}Always{% else %}IfNotPresent{% endif %}
+
+  schedule: "0 7 * * *"
+
+  config:
+    network: "{{ ethereum_network_name }}"
+    discordChannel: "1207978750004690974"
+    ethereumCls:
+      - grandine
+      - lighthouse
+      - lodestar
+      - nimbus
+      - prysm
+      - teku
+    ethereumEls:
+      - besu
+      - erigon
+      - ethereumjs
+      - geth
+      - nethermind
+      - nimbusel
+      - reth
```

Comment:
Is there a better way to get a pre-defined list of CL/EL's for the devnet in question?

Reply:
cc @skylenet

Reply:
our ansible template always has this variable `consensus_node` and `execution_node`. You could use it as `groups['execution_node']` and `groups['consensus_node']. would def let @skylenet check if there's a smarter way tho.

Reply:
Thanks Pari. [301d27a](https://github.com/ethpandaops/ansible-collection-general/pull/302/commits/301d27aa294a726946e0a580383d8a47f2eb9003)

Now utilising the ansible var `groups`, so its driven by the inventory structure. Still not 💯 on this, as I do need to use a regex to match/filter.